### ISSUE TITLE
toBytes method in handler.php

### ISSUE THREAD

**Issue by AdityaSuripeddi1992** [FineUploader/php-traditional-server#22]
I would like to know if this switch case in toBytes method of handler is correct when it is m (or g) we are multiplying $val by only 1024 and not (1024)*(1024)

switch($last) {
            case 'g': case 'G': $val *= 1024;
            case 'm': case 'M': $val *= 1024;
            case 'k': case 'K': $val *= 1024;
        }

I apologize in case of any mistakes in my observations

**Comment by MCF** [comment]
Yes the switch statement in the toBytes method is correct.  I've recently refactored the function in a pending pull request.  I renamed a few variables and rearranged the format of the switch statement, you may find the logic a little clearer in that version:

https://github.com/FineUploader/php-traditional-server/pull/20/files#diff-e354cf92d749c9adff9f282fdf93c192L343

I'm not sure if I understand your concert with the partSize and chunking though.  I don't think it would be impacted by the toBytes function working or not.  The only role that function plays is in an error check at the start of the handleUpload function that confirms if your PHP settings are acceptable given your server size setting for the file size limit (the sizeLimit setting in the handler).

If you have set a sizeLimit in the PHP upload handler and the toBytes function were to cause an error you would receive an error response of `Server error. Increase post_max_size and...`.

BTW the error returns from this PHP sample can sometimes be a little unclear.  Sometimes the http response from the server is set to 200, but the contents of the response is actually an error.  It needs to be cleaned up a bit.  Be sure to inspect the response contents of any http responses if you are running into difficulties.

**Comment by AdityaSuripeddi1992** [comment]
thank you @MCF  for  response I totally forgot the abscence of break statement in switch case, I was able to clear the chunking error which was due to wrong command used to link with php.ini file .
